modular_citadel/code/modules/arousal/organs/breasts.dm

```diff
@@ -6,15 +6,20 @@
 	zone 					= "chest"
 	slot 					= "breasts"
 	w_class 				= 3
-	size 					= BREASTS_SIZE_DEF
+	size 					= BREASTS_SIZE_DEF  //SHOULD BE A LETTER, starts as a number...???
+	var/cached_size			= null //for enlargement SHOULD BE A NUMBER
+	var/prev_size			//For flavour texts SHOULD BE A LETTER
+	var/breast_sizes 		= list ("a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "huge", "flat")
```

Comment:
As I mentioned in Discord, there's already a global list of sizes 

https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/code/__HELPERS/global_lists.dm#L56

Reply:
All this time I thought GLOB performed like python's glob.